hw/ip/otbn/dv/model/otbn_trace_checker.h

```diff
@@ -75,6 +75,10 @@ class OtbnTraceChecker : public OtbnTraceListener {
   // through MatchPair if there is any.
   const OtbnIssTraceEntry::IssData *PopIssData();
 
+  // Tell the model to not execute checks to see if secure wiping has written
+  // random data to all registers before wiping them with zeroes if set.
```

Comment:
That's the description of the flag, rather than the function. Maybe something like this would be better? (Please re-flow! I typed it in GitHub!)
```suggestion
  // Tell the model not to execute checks to see if secure wiping has written
  // random data to all registers before wiping them with zeroes on the next
  // secure wipe entry.
```